### PR TITLE
USWDS-Site: Summary box accessibility checklist

### LINKED ISSUE #2903: USWDS-Site: Publish summary box accessibility checklists page
## Summary
Publish the `usa-summary-box` accessibility test checklist items found in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc :lock:). 

The process for creating a new accessibility test page can be found in the [Creating accessibility checklists - process doc](https://docs.google.com/document/d/14WTCsdzgwpoz_Miztd66RmEGfwwSEUFCVN1DkSQITTE/edit#heading=h.fa3sv1ray42g) (Google Doc :lock:).

### PR DESCRIPTION
**Pull request by mlloydbixal**
# Summary

Added accessibility test page for summary box component.

> [!Important]
> We need to update changelog dates before merge

## Related issue

Closes #2903 

## Preview link
[Summary Box Component Page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/ml-summary-box-checklist/components/summary-box/)
[Summary Box Tests Page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/ml-summary-box-checklist/components/summary-box/accessibility-tests/)

## Testing and review
Follow these steps:

1. Confirm that both the main component page and the accessibility tests page have the correct compliance tag at the top.
2. Confirm that the main component page links to the accessibility tests page in the side navigation.
3. Confirm that the main component page links to the accessibility tests page from the accessibility guidance section.
4. Confirm that the main component page shows the accessibility tests summary.
5. Check that accessibility tests page provides the correct counts for each test status type.
6. Confirm the test checklist summaries and data are accurate.
7. Confirm no visual issues.
8. Confirm there is an appropriate changelog entry on both the main component page and the accessibility tests page.

### CODE REVIEW

File: _data/changelogs/component-summary-box.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@ changelogURL:
 # Add a list of changelog items with newest items first. Uncomment the 'items' data field when adding the first item.
 
 items:
+  - date: NNNN-NN-NN
+    summary: Added WCAG compliance tag and accessibility test status section.
+    affectsGuidance: true
+    githubPr: 2903
```
Comment (amyleadem) [inline _data/changelogs/component-summary-box.yml:11]: **issue:** This should point to this PR

```suggestion
    githubPr: 3016
```

File: _data/changelogs/component-summary-box-accessibility.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title: Summary box accessibility tests
+type: component
+items:
+  - date: NNNN-NN-NN
+    summary: Added accessibility tests page.
+    affectsGuidance: true
+    githubPr: 2903
```
Comment (amyleadem) [inline _data/changelogs/component-summary-box-accessibility.yml:7]: **issue:** This should point to this PR

```suggestion
    githubPr: 3016
```

File: _components/summary-box/accessibility-tests.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/summary-box/accessibility-tests/
+layout: accessibility-test
+component:
+ name: Summary box
+title: Summary box accessibility tests
+category: Components
+lead: Any USWDS Summary box component should pass these manual accessibility tests.
```
Comment (amyleadem) [inline _components/summary-box/accessibility-tests.md:8]:
```suggestion
lead: Any USWDS summary box component should pass these manual accessibility tests.
```

File: _data/accessibility-tests/summary-box.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  # Keyboard navigation tests
+  - summary: Keyboard actions can access all summary box functionality.
+    summary_additional: You can engage with any links provided in a summary box and perform all other functions in the component using only your keyboard.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1
+  # Screen reader tests
+  - summary: Screen reader announces summary box region.
+    summary_additional: When using a screen reader to access the summary box, you hear ""summary box"" (or region in JAWS).
```
Comment (amyleadem) [inline _data/accessibility-tests/summary-box.yml:51]: **issue:** Typo

```suggestion
    summary_additional: When using a screen reader to access the summary box, you hear "summary box" (or region in JAWS).
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
Looking good! Just a few minor changes needed. Thanks @mlloydbixal

File: _data/accessibility-tests/summary-box.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.4
+  - summary: All headings are clear and descriptive.
+    summary_additional: When you view the summary box, the headings are clear and concise.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.6
```
Comment (mahoneycm) [inline _data/accessibility-tests/summary-box.yml:28]: **Question:** We're listing `2.4.6` in General & Screen Reader tests. Do we need both? Do we list it twice in other tests?

I'm thinking we should compare to other components that list `2.4.6` and pick the one that's more common/logical for this component

**Review by mahoneycm — CHANGES_REQUESTED** [review]
Noticed a test appearing twice. Other than what Amy mentioned, everything else is looking solid!

File: _data/accessibility-tests/summary-box.yml (hunk 5 above)
Comment (mlloydbixal) [reply _data/accessibility-tests/summary-box.yml:28]: If I understand what you mean, accordion and table both list 2.4.6 more than once

Comment (mlloydbixal) [reply _data/accessibility-tests/summary-box.yml:28]: Updated in [7f309b3](https://github.com/uswds/uswds-site/pull/3016/commits/7f309b3a5923de678481e29da05bc1f692a628df) to remove screen reader test in favor of the general one

File: _data/changelogs/component-summary-box.yml (hunk 1 above)
Comment (mlloydbixal) [reply _data/changelogs/component-summary-box.yml]: Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)

File: _data/changelogs/component-summary-box-accessibility.yml (hunk 2 above)
Comment (mlloydbixal) [reply _data/changelogs/component-summary-box-accessibility.yml]: Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)

File: _components/summary-box/accessibility-tests.md (hunk 3 above)
Comment (mlloydbixal) [reply _components/summary-box/accessibility-tests.md]: Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)

File: _data/accessibility-tests/summary-box.yml (hunk 4 above)
Comment (mlloydbixal) [reply _data/accessibility-tests/summary-box.yml]: Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)

**Review by amyleadem — APPROVED** [review]
LGTM! Thanks @mlloydbixal

**Review by mahoneycm — APPROVED** [review]
LGTM! Thanks Mandy!

**Review by annepetersen — DISMISSED** [review]
LGTM

**Comment by annepetersen** [comment]
Not sure whose commits weren't signed, but looks like that's our current blocker 👀

**Comment by mlloydbixal** [comment]
@annepetersen it was me! Didn't realize but I am officially verified, however it looks like it's only verifying my commits moving forward?

**Review by annepetersen — APPROVED** [review]

**Pull request merged**